src/clusterfuzz/_internal/issue_management/google_issue_tracker/issue_tracker.py

```diff
@@ -68,6 +70,24 @@ def _get_access_limit_from_labels(labels: issue_tracker.LabelStore):
     return None
 
 
+def retry_on_invalid_gaia_accounts(func):
+  """Decorator to retry saving an issue, skipping CCs
+    without a GAIA account."""
+
+  @functools.wraps(func)
+  def wrapper(self, *args, **kwargs):
+    try:
+      return func(self, *args, **kwargs)
+    except Exception as e:
+      if "HttpError 400" in str(e):
+        emails_to_skip = re.findall(r'[\w\.-]+@[\w\.-]+', str(e))
```

Comment:
Do multiple emails get listed in the exception? Or is it one at a time?

Reply:
I have no idea to be honest, I will do some code archeology on buganizer and get back to you.

Reply:
ack. my main question is if we need to run this in a loop up to a certain number of retries.

Reply:
No, one retry is enough. Buganizer concatenates all failed emails, linked source reference in internal chat